core/server/api_container/server/api_container_service.go

```diff
@@ -774,3 +778,12 @@ func getTextRepresentation(reader io.Reader, lineCount int) (*string, error) {
 	text := textRepresentation.String()
 	return &text, nil
 }
+
+func convertServiceStatusToServiceInfoStatus(serviceStatus service.ServiceStatus) kurtosis_core_rpc_api_bindings.ServiceStatus {
+	switch serviceStatus {
+	case service.ServiceStatus_Started:
+		return kurtosis_core_rpc_api_bindings.ServiceStatus_RUNNING
+	default:
```

Comment:
This is an errorprone pattern. If we add more values to the enum in the future we will silently default to STOPPED.
We should rather `case` on the two values and throw an error for anything else, this way we will fail loudly. Ideally we would want to fail at compile time, but go doesn't make that super easy. Less ideally but still better than throwing an error would be to have a good linter for this, but not sure if that exists. Short term throwing an error is good enough